pkg/wwan/mmagent/mmdbus/client.go

```diff
@@ -599,6 +605,7 @@ func (c *Client) getModemStatus(modemObj dbus.BusObject) (
 				//             Num apps: 0
 				//             Is eUICC: no
 				// TODO: should we call mbimcli/qmicli ?
+				simCard.Type = types.SimTypePhysical
```

Comment:
This change is not described in the commit message, right? I think it deserves a mention along with the description about what was changed...

Reply:
Added comment about this change under the second commit description.